pkg/test/framework/components/istio/operator.go

```diff
@@ -702,6 +692,30 @@ func configureDirectAPIServiceAccessForCluster(ctx resource.Context, env *kube.E
 	return nil
 }
 
+func (i *operatorComponent) createServiceAccount(ctx resource.Context, cluster resource.Cluster) error {
+	// TODO(landow) we should not have users do this. instead this could be a part of create-remote-secret
```

Comment:
+1. @nmittler ?

Reply:
+1 ... do you want to just make that change in this PR?